### PR TITLE
Expand available parameters for windows_task

### PR DESCRIPTION
**Pull request by Hubert-Mach**

### CODE REVIEW

**Comment by clintoncwolfe** [comment]
Thank you for your contribution! However, before we can accept it, we ask that you sign your commits, to indicate that you are the author of the code and that it otherwise meets the terms of our [Developer Certification of Origin](https://github.com/inspec/inspec/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco) policy.

After you have read the DCO, please sign your code, as follows:
```
git commit --amend -s
git push -f
```

Thank you!

**Comment by clintoncwolfe** [comment]
Thank you for the contribution! However, your code currently does not pass our 
coding standard, which is enforced by RuboCop. If RuboCop does not like the code,
it will fail CI tests, such as TravisCI and Appveyor.

To check your code locally, run:
```
$ bundle exec rubocop
```

To auto-correct, run:
```
$ bundle exec rubocop -a
```

Thank you!

File: lib/resources/windows_task.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class WindowsTasks < Inspec.resource(1)
     name 'windows_task'
     supports platform: 'windows'
     desc 'Use the windows_task InSpec audit resource to test task schedules on Microsoft Windows.'
-    example <<~EXAMPLE
```
Comment (clintoncwolfe) [inline lib/resources/windows_task.rb]: We have standardized on `<<~` because it fixes indentation errors in other documentation formats.  Please revert this change.

File: lib/resources/windows_task.rb [hunk 2]
```diff
@@ -61,7 +61,99 @@ def task_to_run
     def run_as_user
       info[:run_as_user]
     end
+    
+    def hostname
```
Comment (clintoncwolfe) [inline lib/resources/windows_task.rb:65]: It seems like this might be a good candidate for dynamically defining these methods.

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Hi! Thanks for your contribution! It looks fairly straightforward, and we appreciate including docs right away.  

It would be good, however, to try to include some kind of testing, unit or integration. Let us know if you need help with that.

File: lib/resources/windows_task.rb (hunk 1 above)
Comment (Hubert-Mach) [reply lib/resources/windows_task.rb]: Hi!
I'd need some help. Found windows_task_test.rb for unit tests and reviewed helper file however don't know how windows task is mocked and what values should I expect.

<<~EXAMPLE is reverted.

Comment (Hubert-Mach) [reply lib/resources/windows_task.rb]: Never mind. Done :)

**Review by miah — APPROVED** [review]

**Comment by kekaichinose** [comment]
Per InSpec team [working agreement](https://github.com/inspec/inspec/blob/master/inspec_team-working_agreement.md), this issue has not been addressed for more than 3 business days, and will be closed.

If you feel this was done in error, please reopen the issue and address any feedback in the comments so we can continue this forward in the workflow.

Thank you!
__Chef InSpec Team__
